lib/diagnostic.md

```diff
@@ -54,7 +76,9 @@ Please write a method that traverses from the lister DOM node (one of the list
 item elements) to the body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+```
+$('listerItem').hide()
+```
```

Comment:
$('.listerItem').parent().parent().hide().  I think you misunderstood what I was asking